src/main/java/no/unit/nva/model/contexttypes/Degree.java

```diff
@@ -0,0 +1,71 @@
+package no.unit.nva.model.contexttypes;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.Level;
+
+import java.util.List;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class Degree extends Book {
+
+    public Degree() {
+        super();
+    }
+
+    private Degree(Builder builder) {
+        super();
+        setSeriesTitle(builder.seriesTitle);
+        setSeriesNumber(builder.seriesNumber);
+        setLevel(builder.level);
+        setOpenAccess(builder.openAccess);
+        setPeerReviewed(builder.peerReviewed);
+        setIsbnList(builder.isbnList);
+    }
+
+
```

Comment:
Just in case implement an equals and hashcode that call the methods of the parent class

Reply:
This ends logically the same as super.equals(object), which is what we are calling